experimental/generation/generator/packages/library/templates/generator.lg

```diff
@@ -297,9 +297,22 @@ ${'#'} ${property}Entity_getValueText(val)
 {
     "$schema": "${appSchema}",
     "$kind": "Microsoft.OnEndOfActions",
-    "condition":"=!$${property} || $PropertyToChange == '${property}'",
-    "priority": "=indexOf(dialog.requiredProperties, '${property}')",
-    "actions": [       
+    "condition":"=!$${property} || $PropertyToChange == '${property}'", 
+    "priority": "=lib.missingPriority(dialog.requiredProperties, '${property}')",
```

Comment:
>dialog.requiredProperties,  [](start = 38, length = 27)

Build reference into lib.missingPriority.

Reply:
do you mean that "priority" is kind of like lib.missingPriority('${property}')"  and the missingPriority function as follows:
# missingPriority(property)
- ${indexOf(dialog.requiredProperties, property)}